includes/shortcodes/class-wc-shortcode-cart.php

```diff
@@ -40,7 +40,9 @@ public static function calculate_shipping() {
 			}
 
 			if ( $address['country'] ) {
+				if ( ! WC()->customer->get_billing_first_name() ) {
 				WC()->customer->set_billing_location( $address['country'], $address['state'], $address['postcode'], $address['city'] );
```

Comment:
Not correctly indented. Missing a tab here.

Reply:
Thanks Claudio! I just added it...